examples/treegrid/treegrid-1.html

```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-expanded=<q>true</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li>Indicates the <code>row</code> can show or hide other message rows within this <code>treegrid</code>.</li>
+              <li><code>aria-expanded=<q>true</q></code> indicates the message rows that responded to this e-mail are visible.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>false</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
+              <li>When the <code>row</code> is visible (e.g. a top level message or a message whose parent message is expanded), <code>aria-hidden=<q>false</q></code> makes the message row information available to assistive technologies.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>true</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
+              <li>When a message <code>row</code> is <strong>not</strong> visible (e.g. is parent message is collapsed), <code>aria-hidden=<q>true</q></code> <strong>hides</strong> the message row  information from assistive technologies.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-level=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li>The level of the e-mail message identifies a set of messages that share responding to the same parent message row (e.g. same subject line and sender).</li>
+              <li>Top level messages have <code>aria-level=<q>1</q></code>.</li>
+              <li>The <code>aria-level</code> value for other message rows is defined by increasing the level by one over the <code>aria-level</code> value of the parent message.  For example, message rows responding to a top level message have <code>aria-level=<q>2</q>.</li>
+              <li>NOTE: Message rows of the same level and sharing the same parent e-mail are shown or hidden as a group when the parent message row is expanded or contracted.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-setsize=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            Identifies the number of message rows that responded to the parent e-mail message.
```

Comment:
Recommend the following wording to be consistent with the tree examples:

Defines the number of rows in the set of rows that are in the same branch and at the same level within the hierarchy.